core/.changelog.d/2841.changed

```diff
@@ -0,0 +1 @@
+Reboot to bootloader does not require experimental features
```

Comment:
changelog-wise, this is a "new" feature. so please move this to `added` and modify the text accordingly

Reply:
Done in the force-push